TM1py/Services/ElementService.py

```diff
@@ -142,7 +141,27 @@ def get_elements_by_level(self, dimension_name: str, hierarchy_name: str, level:
             "/api/v1/Dimensions('{}')/Hierarchies('{}')/Elements?$select=Name&$filter=Level eq {}",
             dimension_name,
             hierarchy_name,
-            level)
+            str(level))
+        response = self._rest.GET(url, **kwargs)
+        return [e["Name"] for e in response.json()['value']]
+
+    def get_elements_by_wildcard(self, dimension_name: str, hierarchy_name: str,
```

Comment:
I would suggest calling this function `get_elements_filtered_by_wildcard`. Then it is "in line" with the  `get_elements_filtered_by_attribute` function in the same class.
What do you think?

Reply:
I actually wanted that name and then thought it wouldn't be "in line" with the `get_elements_by_level` method. I will update the name.

Reply:
Well that's a good point. Then I guess both names are fine :)